crates/ruff_cli/src/printer.rs

```diff
@@ -601,7 +602,7 @@ fn print_message<T: Write>(
                 source: &source.contents,
                 line_start: message.location.row(),
                 annotations: vec![SourceAnnotation {
-                    label: message.kind.rule().code(),
+                    label: &label,
```

Comment:
Removed from inline just for clarity? Or some other reason?

Reply:
Removed from inline due to borrow checking reasons. Otherwise borrowchk complains due to returning a temporary reference to an owned value.